app/jobs/google/search_keyword_job.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Google
+  class SearchKeywordJob < ApplicationJob
+    queue_as :default
+
+    def perform(keyword)
+      search_result = Google::SearchService.new(keyword[:name]).call
```

Comment:
Usually when passing data to the background job, we pass the ID instead of the whole object. 

```suggestion
    def perform(keyword_id)
      keyword = Keyword.find(keyword_id)
      search_result = Google::SearchService.new(keyword.name).call
```

Reply:
This is more than just a convention: Background jobs are enqueues (e.g. redis will store the reference to the job and the arguments passed into it). 
To enqueue a job, the arguments are then serialized and stored.
Passing the whole object can have multiple issues:
- slow queue performance as the enqueued jobs are bigger
- serialize/deserialize might miss the associated objects
- the state of the object in DB might have changed at the time the job is running
- ...